src/luarocks/admin/cmd/remove.lua

```diff
@@ -68,6 +65,17 @@ local function remove_files_from_server(refresh, rockfiles, server, upload_serve
    util.printout("Updating index.html...")
    index.make_index(local_cache)
 
+   if protocol == "file" then
+       local cmd = cfg.variables.RSYNC.." "..cfg.variables.RSYNCFLAGS.." --delete "..local_cache.."/ ".. server_path.."/"
```

Comment:
@0x501D  After merging I realized that the `file` protocol mode still uses the RSYNC command. Would it be possible to just do this using `fs.delete`?...

Reply:
I think I can do this